src/hooks/useVideoInfo.js

```diff
@@ -0,0 +1,36 @@
+import {useState, useEffect} from 'react';
+import youtube from '../apis/youtube';
+
+const useVideoInfo = (videoID) => { 
+    
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  
+  const parseData = (video) => {
+    const title = video.title;
+    const description = video.description;
+
+    return [title, description];
+  }
+
+  const handleSubmit = async (videoID) => {
+    const response = await youtube.get('/videos', {
+      params: {
+        id: videoID
+      }
+    })
+    
+    const [title, description] = parseData(response.data.items[0].snippet);
+    setTitle(title);
+    setDescription(description);
+  }
+
+  useEffect(()=>{
+    handleSubmit(videoID)
+  })
```

Comment:
I left it out because with it it sends this warning
React Hook useEffect has missing dependencies: 'handleSubmit' and 'videoID'. Either include them or remove the dependency array  react-hooks/exhaustive-deps